<html xmlns:py="http://genshi.edgewall.org/" xmlns:xi="http://www.w3.org/2001/XInclude" py:strip="">


<py:def function="announcement_id">
PMASA-2007-6
</py:def>

<py:def function="announcement_date">
2007-10-17
</py:def>

<py:def function="announcement_updated">
2007-10-24 (CVE reference)
</py:def>


<py:def function="announcement_summary">
XSS vulnerabilities
</py:def>


<py:def function="announcement_description_fmt">
<p>
We received an advisory from Omer Singer, The DigiTrust Group, and we wish to thank him for his work. It was possible to trigger this attack on server_status.php.
</p>
<p>
Our team fixed also other possible XSS vulnerabilities regarding PHP_SELF, PATH_INFO, REQUEST_URI.
</p>
</py:def>


<py:def function="announcement_severity">
We consider these vulnerabilities to be serious.
</py:def>


<py:def function="announcement_affected">
Probably all versions before 2.11.1.2. 
</py:def>


<py:def function="announcement_solution">
Upgrade to phpMyAdmin 2.11.1.2 or newer.
</py:def>


<py:def function="announcement_references">
<a href="http://www.digitrustgroup.com/advisories/TDG-advisory071015a.html">http://www.digitrustgroup.com/advisories/TDG-advisory071015a.html</a>
</py:def>

<py:def function="announcement_cve">
CVE-2007-5589
</py:def>

<py:def function="announcement_patches">
Patches are referenced here:
<a href="http://phpmyadmin.svn.sourceforge.net/viewvc/phpmyadmin?view=rev&amp;revision=10796">http://phpmyadmin.svn.sourceforge.net/viewvc/phpmyadmin?view=rev&amp;revision=10796</a>
</py:def>


<xi:include href="_page.tpl" />
</html>
